db/seeds.rb

```diff
@@ -6,23 +6,30 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-categories = Category.create([{title: "Apparel"},{title: "Snowboards"}])
+categories = Category.create( [ { title: "Apparel" },{ title: "Snowboards" } ] )
 
-apparel = Category.first.items.create([{name: "Hoody",
-                      description: "Keeps you slightly warm",
-                      price: 80},
+apparel = Category.first.items.create( [
+                  { name: "Hoody",
+                    description: "Keeps you slightly warm",
+                    price: 80,
+                    image: File.open("app/assets/images/hoody.jpg") },
 
-                     {name: "Sweet Jacket",
-                      description: "Keeps you warm",
-                      price: 240}])
+                  { name: "Sweet Jacket",
+                    description: "Keeps you warm",
+                    price: 240,
+                    image: File.open("app/assets/images/jacket.jpg") } ] )
 
 
-snowboards = Category.last.items.create([
-                    {name: "gwar possum",
-                     description: "a snowboard for gwar concerts",
-                     price: 15},
+snowboards = Category.last.items.create( [
+                      { name: "gwar possum",
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.